src/renderer/viz/expressions/viewportFeatures.js

```diff
@@ -64,7 +67,7 @@ export default class ViewportFeatures extends BaseExpression {
         this.type = 'featureList';
         this._isViewport = true;
         this._requiredProperties = properties;
-        this._FeatureProxy = null;
+        this._columns = null;
```

Comment:
columns is SQL terminology, I don't think it should be named like that here

Reply:
Thx! I was keeping the original name from LIN 91, but I think I'm gonna rename it to "_propertyNames"